components/Portfolio/Trading/Overview/PositionGraph/index.tsx

```diff
@@ -120,12 +121,12 @@ const PositionGraph: FC<PGProps> = styled(
                         </Row>
                         <Row>
                             <BorderlessCell>
-                                <Button className="primary" onClick={closePosition}>
-                                    Close Position
-                                </Button>
+                                <Button onClick={closePosition}>Close Position</Button>
                             </BorderlessCell>
                             <BorderlessCell inner>
-                                <Button className="primary">Adjust Position</Button>
+                                <Link href="/">
+                                    <Button>Adjust Position</Button>
+                                </Link>
```

Comment:
Assume these are meant to no longer be primary

Reply:
yes